src/app/home/home.component.ts

```diff
@@ -87,7 +92,22 @@ export class HomeComponent implements OnInit {
     }
   ];
 
-  constructor(private sanitizer: DomSanitizer) {}
+  constructor(private configService: ConfigService, private sanitizer: DomSanitizer) {}
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.configService.getConfig().subscribe((config: Config) => {
+      if (config != null) {
+        this.discourseUrl = this.configService.getDiscourseURL();
+        this.discourseNewsCategory = this.configService.getDiscourseNewsCategory();
+        if (
+          this.discourseUrl != null &&
+          this.discourseUrl !== '' &&
+          this.discourseNewsCategory != null &&
+          this.discourseNewsCategory !== ''
+        ) {
+          this.isDiscourseNewsAvailable = true;
+        }
```

Comment:
I think this statement can be written a bit more cleanly, something like this?

```suggestion
        this.isDiscourseNewsAvailable =
          this.isSet(this.discourseUrl) && this.isSet(this.discourseNewsCategory);
          ...
   }
        
        private isSet(str: string): boolean {
          return str != null && str !== '';
        }
```